Add an example of webrtc-direct (#868)
libp2p/js-libp2p: examples/webrtc-direct/README.md

### About
An example that uses [js-libp2p-webrtc-direct](https://github.com/libp2p/js-libp2p-webrtc-direct) for connecting a
nodejs libp2p and a browser libp2p clients. To run the example:

## 0. Run a nodejs libp2p client
When in the root folder of this example, type `node node.js` in terminal. You should see an address that listens for
incoming connections. Below is just an example of such address. In your case the suffix hash will be different.
```bash
$ node node.js 
Listening on:
/ip4/127.0.0.1/tcp/9090/http/p2p-webrtc-direct/p2p/QmUKQCzEUhhhobcNSrXU5uzxTqbvF1BjMCGNGZzZU14Kgd
```

> ## 1. Prepare a browser libp2p client

Reviewer: ```suggestion
## 1. Prepare a browser libp2p dialer
```

> Copy-paste the above address into the field `list` in `public/browser.js`:

Reviewer: ```suggestion
Copy-paste the above address into the field `list` in `public/dialer.js`:
```

```js
    peerDiscovery: {
      [Bootstrap.tag]: {
        enabled: true,
        // paste the address into `list`
        list: ['/ip4/127.0.0.1/tcp/9090/http/p2p-webrtc-direct/p2p/QmUKQCzEUhhhobcNSrXU5uzxTqbvF1BjMCGNGZzZU14Kgd']
      }
    }
```

> ## 2. Run a browser libp2p client

Reviewer: ```suggestion
## 2. Run a browser libp2p dialer
```

### Webrtc-direct example

An example that uses [js-libp2p-webrtc-direct](https://github.com/libp2p/js-libp2p-webrtc-direct) for connecting a

> nodejs libp2p and a browser libp2p clients. To run the example:

Reviewer: ```suggestion
nodejs libp2p and browser libp2p clients. To run the example:
```

## 0. Run a nodejs libp2p listener

When in the root folder of this example, type `node listener.js` in terminal. You should see an address that listens for
incoming connections. Below is just an example of such address. In your case the suffix hash (`peerId`) will be different.

```bash
$ node listener.js 
Listening on:
/ip4/127.0.0.1/tcp/9090/http/p2p-webrtc-direct/p2p/QmUKQCzEUhhhobcNSrXU5uzxTqbvF1BjMCGNGZzZU14Kgd
```

## 1. Prepare a browser libp2p dialer

> Copy-paste the above address into the field `list` in `public/dialer.js`:

Reviewer: ```suggestion
Confirm that the above address is the same as the field `list` in `public/dialer.js`:
```

```js
    peerDiscovery: {
      [Bootstrap.tag]: {
        enabled: true,
        // paste the address into `list`
        list: ['/ip4/127.0.0.1/tcp/9090/http/p2p-webrtc-direct/p2p/QmUKQCzEUhhhobcNSrXU5uzxTqbvF1BjMCGNGZzZU14Kgd']
      }
    }
```

## 2. Run a browser libp2p dialer
When in the root folder of this example, type `npm run dev` in terminal. You should see an address where you can browse

> the running client. Open this address in your browser. In my case the address was `http://127.0.0.1:12345`. In console

Reviewer: ```suggestion
the running client. Open this address in your browser. In console
```
